### ISSUE TITLE
PySide build broken on MacOS X

### ISSUE THREAD

**Issue by astrofrog** [ContinuumIO/anaconda-issues#1132]
The pyside 1.1.2 py27_1 appears to be severely broken on Mac:

```
In [1]: from PySide import QtGui

In [2]: QtGui.QApplication([])
Qt internal error: qt_menu.nib could not be loaded. The .nib file should be placed in QtGui.framework/Versions/Current/Resources/  or in the resources directory of your application bundle.
Abort trap: 6
```

**Comment by ccordoba12** [comment]
Nobody has touched PySide in two years or more :-)

**Comment by astrofrog** [comment]
@ccordoba12 - nevertheless the failure is real, and several users have reported this to me. Could it be due to the update of one of PySide's dependencies?

**Comment by astrofrog** [comment]
Here's a minimal example failing on Travis: https://travis-ci.org/astrofrog/debug-pyside/builds/164895121

**Comment by ccordoba12** [comment]
Yes, it seems due to our latest Qt4 build (but I don't why that caused the breakage).

**Comment by astrofrog** [comment]
Any updates on this? Would it make sense to pin PySide to previous Qt4 versions?

**Comment by ccordoba12** [comment]
Yes, nobody is going to update PySide anytime soon in Continuum. Or you could just use the conda-forge packages :-)
